test_schemas/en/test_checkbox.json

```diff
@@ -32,6 +32,67 @@
             "groups": [
                 {
                     "blocks": [
+                        {
```

Comment:
I don't think adding new blocks in test schemas before existing ones is a good thing. It means other un-related tests need to be updated to add the new block. I think we should add a new schema, and have the three different variants and their tests - default label (select all that apply), no label and custom label.